lib/workers/branch/schedule.js

```diff
@@ -10,6 +11,17 @@ function fixShortHours(input) {
   return input.replace(/( \d?\d)((a|p)m)/g, '$1:00$2');
 }
 
+function hasValidTimezone(timezone) {
+  if (timezone) {
+    let message;
```

Comment:
This `message` variable is not necessary

Reply:
removed